.github/workflows/functions.yml

```diff
@@ -3,7 +3,7 @@ name: functions
 on:
   pull_request:
     paths:
-    - 'FirebaseFunctions/**'
+    - 'FirebaseFunctions**'
```

Comment:
This will now trigger with podspec updates - I thought we didn't want that for some reason but I can't remember why (and I can't think of why we wouldn't want it triggering...). Just wanted to call it out to see if you remembered / knew.

Reply:
Yes, we should run tests if podspec's change. However, the new `pod spec` tests require consistent versions - so I'm changing to keep the 9.0.0 only pods consistent with the rest for now.

Reply:
Actually we can't since the `pod spec lint` is only for master. I disabled and created #9313 to track.